internal/adapter/lsp/server.go

```diff
@@ -104,6 +104,12 @@ func NewServer(opts ServerOpts) *Server {
 			ResolveProvider:   boolPtr(true),
 		}
 
+		capabilities.ReferencesProvider = &protocol.ReferenceOptions{
+			WorkDoneProgressOptions: protocol.WorkDoneProgressOptions{
+				WorkDoneProgress: boolPtr(false),
+			},
+		}
```

Comment:
These are the default values so we can skip it.
```suggestion
		capabilities.ReferencesProvider = &protocol.ReferenceOptions{}
```